### PR TITLE
OKTA-303663: Update the visual style and copy for /code pages

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## Description:
- **What's changed?** 
  * 'Create Free Account' button is removed from /code pages
  * Github samples links are replaced with 'Sample App' button
- **Is this PR related to a Monolith release?** 
No
- **Netlify build** 
https://5fb2843537c67bd46d5deab7--reverent-murdock-829d24.netlify.app/code/dotnet/aspnet/
### Resolves:

* [OKTA-303663](https://oktainc.atlassian.net/browse/OKTA-303663)

### CODE REVIEW

**Comment by oleksandrpravosudko-okta** [comment]
@alexdahl-okta, @albertchen-okta - looking for review for this update - thanks

**Comment by oleksandrpravosudko-okta** [comment]
@alexdahl-okta - looking for feedback on dropdown button [implementation](https://5fb2843537c67bd46d5deab7--reverent-murdock-829d24.netlify.app/code/dotnet/aspnet/)

**Comment by alexdahl-okta** [comment]
@oleksandrpravosudko-okta, dropdown button looks good to me – thanks!

File: packages/@okta/vuepress-theme-prose/global-components/DropdownButtonOption.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class='dropdown-button-option-value'>
+  <a href="/">
```
[the remark this answers is not in the document]
Comment (oleksandrpravosudko-okta) [reply packages/@okta/vuepress-theme-prose/global-components/DropdownButtonOption.vue]: thanks - `href` prop should be used indeed

File: packages/@okta/vuepress-theme-prose/global-components/DropdownButton.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <div :class="{'dropdown-button': true, 'dropdown-button-expanded': isExpanded}" @click="isExpanded = !isExpanded">
+    <a class='Button--blueDarkOutline' data-proofer-ignore>
```
[the remark this answers is not in the document]
Comment (oleksandrpravosudko-okta) [reply packages/@okta/vuepress-theme-prose/global-components/DropdownButton.vue]: this was a shortcut to get some of [existing styles](https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/assets/css/components/_codePages.scss#L34) for free - replaced with block element

**Review by AndriiDovhopolov-okta — APPROVED** [review]

**Pull request merged**